backend/src/test/java/com/redhat/cloud/notifications/events/LifecycleITest.java

```diff
@@ -669,6 +675,31 @@ private void checkEventTypeBehaviorGroups(Header identityHeader, String eventTyp
         }
     }
 
+    private void checkEventTypeBehaviorGroupsV2(Header identityHeader, String eventTypeId, String... expectedBehaviorGroupIds) {
+        String responseBody = given()
+                .basePath(API_NOTIFICATIONS_V_2_0)
+                .header(identityHeader)
+                .pathParam("eventTypeId", eventTypeId)
+                .when()
+                .get("/notifications/eventTypes/{eventTypeId}/behaviorGroups")
+                .then()
+                .statusCode(200)
+                .contentType(JSON)
+                .extract().body().asString();
+
+        // The response body contains the meta information about the history collection. We need to access the
+        // "data" array to access the elements.
+        final JsonObject responseBodyJson = new JsonObject(responseBody);
+        final JsonArray jsonBehaviorGroups = responseBodyJson.getJsonArray("data");
```

Comment:
This test could check the other fields from `responseBodyJson` (`meta` and `links`).

Reply:
Done!